OneCommunityGlobal/HighestGoodNetworkApp
# Xiaoyu - Add editable role info and editable mouseover modal

# Description
![Screenshot 2023-07-19 at 6 40 56 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/de9a994f-b927-4c17-a010-67656b6ad155)
<img width="465" alt="Screenshot 2023-07-25 at 10 53 43 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/65b7066f-b8cd-4adb-ad1d-8e260067be38">
<img width="471" alt="Screenshot 2023-07-25 at 10 54 15 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/10668c8b-df9f-4307-acb0-3479d8db305e">
<img width="481" alt="Screenshot 2023-07-25 at 10 54 31 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/d2268326-c74b-4ece-8dd1-36c1f5527d13">
<img width="596" alt="Screenshot 2023-07-25 at 10 55 08 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/52744242-9040-48e2-a281-7770d5fe6595">
<img width="507" alt="Screenshot 2023-07-25 at 10 56 20 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/0db773fe-08ae-4321-bb1a-26e038a3e337">


## Related PRS (if any):
This frontend PR is related to [#452](https://github.com/OneCommunityGlobal/HGNRest/pull/452) backend PR
…

## Main changes explained:
- Crete editable modal and info components
- create information.js for action, constants, inforeducer, informationService
- Create a readable modal in report page avoid re-render issues (content will changed based on the role info in the permission role info)
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
Login as Owner:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/76c4fc65-98aa-474f-92c6-6aa4dba8f396

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102696148/354ef5e3-855a-43de-967b-14b4cdd75fcb
## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
